### PR TITLE
add arm64 support to netperf image

### PR DESCRIPTION
**Pull request by lubinsz**
I have tested it with Arm64 k8s cluster, the result seems OK.

Signed-off-by: Bin Lu <bin.lu@arm.com>

### CODE REVIEW

**Comment by lubinsz** [comment]
@girishkalele 
Can you check it?
/assign @girishkalele

**Comment by lubinsz** [comment]
cc @BenTheElder

File: hack/build/init-buildx.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
```
Comment (BenTheElder) [inline hack/build/init-buildx.sh]: this script is a hacky experiment coped from the kind repo, I'm not ready to encourage replicating it throughout the project yet.

**Comment by BenTheElder** [comment]
I don't own this, however I am noting that we should probably not copy those scripts from kind yet (also normally when copying code it is good to note [where it came from](https://github.com/kubernetes-sigs/kind/blob/49eba13ba76d76da867e4949486a3b3c315dbda1/hack/build/init-buildx.sh)).
/uncc

**Pull request closed without merging**